code/core/system/proxy-camera/src/ProxyCamera.cpp

```diff
@@ -95,10 +95,11 @@ void ProxyCamera::setUp() {
             const uint32_t WIDTH = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.camera.width");
             const uint32_t HEIGHT = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.camera.height");
             const uint32_t BPP = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.camera.bpp");
-            const uint32_t DEBUG = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.debug");
+            const uint32_t DEBUG = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.camera.debug");
+            const bool FLIPPED = getKeyValueConfiguration().getValue<bool>("proxy-camera.camera.flipped");
```

Comment:
@HangYin2015 Should be

const bool FLIPPED = getKeyValueConfiguration().getValue<uint32_t>("proxy-camera.camera.flipped") == 1;